db/schema.rb

```diff
@@ -22,6 +22,14 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "reviews", force: :cascade do |t|
+    t.string "title", null: false
+    t.text "body", null: false
+    t.integer "rating", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```

Comment:
`reviews` is out of the scope of this PR for user profile photos, next time this should be part of a different PR!

Reply:
This was a pull from master resulting in schema update before we started.